### PR TITLE
tikv-server: check TZ setting

### PR DESCRIPTION
**Pull request by andelf**
related: https://github.com/pingcap/tidb-ansible/pull/20

If TZ is not set, tikv will stat(/etc/localtime) twice every logging operation.

ref: https://stackoverflow.com/questions/4554271/how-to-avoid-excessive-stat-etc-localtime-calls-in-strftime-on-linux

### CODE REVIEW

**Comment by siddontang** [comment]
Do you use strace to check we don't stat too many /etc/localtime ？

File: src/bin/tikv-server.rs [hunk 1]
```diff
@@ -269,6 +270,13 @@ fn check_system_config(config: &toml::Value) {
     for e in util::config::check_kernel() {
         warn!("{:?}", e);
     }
+
+    if cfg!(windows) {
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: You mean `!cfg!(windows)`?

Comment (andelf) [inline src/bin/tikv-server.rs]: 🤣 typo

File: src/bin/tikv-server.rs [hunk 2]
```diff
@@ -269,6 +270,13 @@ fn check_system_config(config: &toml::Value) {
     for e in util::config::check_kernel() {
         warn!("{:?}", e);
     }
+
+    if !cfg!(windows) {
+        if env::var("TZ").is_err() {
+            env::set_var("TZ", "/etc/localtime");
+            error!("environment variable `TZ` is missing, use `/etc/localtime`");
```
Comment (BusyJay) [inline src/bin/tikv-server.rs]: Warning is better.

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**